packages/common/effect/src/jsonPath.ts

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+// TODO(ZaymonFC): Talk with Rich/Josiah about where this module should live.
+
```

Comment:
CC @richburdon, do you have any suggestions for where this should live?

Reply:
fine here for now IMO